README.md

```diff
@@ -90,7 +90,8 @@ Available commands:
 
 The following is a brief explanation of each available command.
 
-* `analyze`: Analyze all relevant files in the current working directory. All engines that are enabled in your `.codeclimate.yml` file will run, one after another. The `-f` (or `format`) argument allows you to set the output format of the analysis (using `json`, `text`, or `html`).
+* `analyze`: Analyze all relevant files in the current working directory. All engines that are enabled in your `.codeclimate.yml` file will run, one after another. The `-f` (or `format`) argument allows you to set the output format of the analysis (using `json`, `text`, or `html`). The `--dev` flag lets you run your own engines during
+development.
```

Comment:
I think `--dev` is under-explained here. I would either explain it more (i.e. "lets you run engines not known to the CLI, for example if you're engine author developing your own, unreleased image"), or not mention it at all.